### ISSUE TITLE
Unexpected error?!

### ISSUE THREAD

**Issue by ma-sadeghi** [spyder-ide/spyder#18753]
### What steps will reproduce the problem?

Not really sure how it happened...................

### Traceback
```python-traceback
Traceback (most recent call last):
  File "/home/amin/mambaforge/lib/python3.9/site-packages/spyder/app/mainwindow.py", line 1318, in plugin_focus_changed
    self.update_search_menu()
  File "/home/amin/mambaforge/lib/python3.9/site-packages/spyder/app/mainwindow.py", line 1445, in update_search_menu
    self.search_menu_actions[3].setEnabled(readwrite_editor)
RuntimeError: wrapped C/C++ object of type SpyderAction has been deleted
Traceback (most recent call last):
  File "/home/amin/mambaforge/lib/python3.9/site-packages/spyder/app/mainwindow.py", line 1318, in plugin_focus_changed
    self.update_search_menu()
  File "/home/amin/mambaforge/lib/python3.9/site-packages/spyder/app/mainwindow.py", line 1445, in update_search_menu
    self.search_menu_actions[3].setEnabled(readwrite_editor)
RuntimeError: wrapped C/C++ object of type SpyderAction has been deleted
```

## Versions

* Spyder version: 5.3.0 
* Python version: 3.9.6
* Qt version: 5.12.9
* PyQt5 version: 5.12.3
* Operating System: Linux 5.17.5-76051705-generic

### Dependencies

```
# Mandatory:
atomicwrites >=1.2.0          :  1.4.0 (OK)
chardet >=2.0.0               :  4.0.0 (OK)
cloudpickle >=0.5.0           :  2.0.0 (OK)
cookiecutter >=1.6.0          :  1.7.3 (OK)
diff_match_patch >=20181111   :  20200713 (OK)
intervaltree >=3.0.2          :  3.0.2 (OK)
IPython >=7.31.1;<8.0.0       :  7.33.0 (OK)
jedi >=0.17.2;<0.19.0         :  0.18.1 (OK)
jellyfish >=0.7               :  0.9.0 (OK)
jsonschema >=3.2.0            :  4.4.0 (OK)
keyring >=17.0.0              :  23.4.0 (OK)
nbconvert >=4.0               :  6.4.4 (OK)
numpydoc >=0.6.0              :  1.2 (OK)
parso >=0.7.0;<0.9.0          :  0.8.3 (OK)
pexpect >=4.4.0               :  4.8.0 (OK)
pickleshare >=0.4             :  0.7.5 (OK)
psutil >=5.3                  :  5.9.0 (OK)
pygments >=2.0                :  2.11.2 (OK)
pylint >=2.5.0                :  2.13.0 (OK)
pyls_spyder >=0.4.0           :  0.4.0 (OK)
pylsp >=1.4.1;<1.5.0          :  1.4.1 (OK)
pylsp_black >=1.2.0           :  1.2.0 (OK)
qdarkstyle >=3.0.2;<3.1.0     :  3.0.2 (OK)
qstylizer >=0.1.10            :  0.2.1 (OK)
qtawesome >=1.0.2             :  1.1.1 (OK)
qtconsole >=5.3.0;<5.4.0      :  5.3.0 (OK)
qtpy >=2.0.1                  :  2.0.1 (OK)
rtree >=0.9.7                 :  0.9.7 (OK)
setuptools >=49.6.0           :  61.0.0 (OK)
sphinx >=0.6.6                :  4.4.0 (OK)
spyder_kernels >=2.3.0;<2.4.0 :  2.3.0 (OK)
textdistance >=4.2.0          :  4.2.2 (OK)
three_merge >=0.1.1           :  0.1.1 (OK)
watchdog >=0.10.3             :  2.1.6 (OK)
xdg >=0.26                    :  0.27 (OK)
zmq >=17                      :  22.3.0 (OK)

# Optional:
cython >=0.21                 :  None (NOK)
matplotlib >=3.0.0            :  None (NOK)
numpy >=1.7                   :  None (NOK)
pandas >=1.1.1                :  None (NOK)
scipy >=0.17.0                :  None (NOK)
sympy >=0.7.3                 :  None (NOK)
```

**Comment by dalthviz** [comment]
Hi @ma-sadeghi thank you for the feedback! Seems like this was already fixed in 5.3.1 (https://github.com/spyder-ide/spyder/issues/17974). The latest Spyder release available is 5.3.2. Could you update and check if the you are able to reproduce the problem after updating? Let us know!

**Comment by ma-sadeghi** [comment]
@dalthviz Thank you for your quick reply. In fact, I don't know what triggered the bug, so can't reproduce it, but I trust you and I'm sure it's already fixed :) So I'm closing this issue.
